app/services/school_search.rb

```diff
@@ -15,6 +19,10 @@ def initialize(query: nil, limit: DEFAULT_LIMIT, lead_schools_only: false)
   end
 
   def call
+    self
```

Comment:
This makes this service quite inconsistent with the others in the app. Couldn't we keep the `ServicePattern` and just return a `SchoolSearchResults` object with the data in?

Reply:
thanks Graham, I've pushed up changes that keep the service pattern and return object